lib/checker.js

```diff
@@ -45,12 +45,62 @@ Checker.prototype.checkFile = function(path) {
 };
 
 /**
+ * Fixes single file.
+ *
+ * @param {String} path
+ * @returns {Promise * Errors}
+ */
+Checker.prototype.fixFile = function(path) {
+    if (!this._configuration.isFileExcluded(path)) {
+        return vowFs.read(path, 'utf8').then(function(data) {
+            var result = this.fixString(data, path);
+            return vowFs.write(path, result.output).then(function() {
+                return result.errors;
+            });
+        }, this);
+    }
+
+    return Vow.resolve(null);
+};
+
+/**
  * Checks directory recursively.
  *
  * @param {String} path
  * @returns {Promise * Error[]}
  */
 Checker.prototype.checkDirectory = function(path) {
+    return this._processDirectory(path, this.checkFile.bind(this));
+};
+
+/**
+ * Checks directory or file.
+ *
+ * @param {String} path
+ * @returns {Promise * Error[]}
```

Comment:
This case is partially covered in JsDoc, so different projects use different notations: `{Promise ~ Type}`, others — `{Promise * Type}`. Following jsdoc documentation, the only close notation is `{Promise<Type>}`.